excel_graph_parser/parser.py

```diff
@@ -0,0 +1,214 @@
+from io import BytesIO
+
+from openpyxl import load_workbook, Workbook
+from openpyxl.cell import Cell
+from viktor import File
+from viktor.external.spreadsheet import SpreadsheetCalculation
+
+import plotly.graph_objects as go
+
+ALLOWED_FIGURE_TYPES = ["lineChart", "scatterChart", "barChart", "pieChart"]
+
+
+class SpreadsheetParser:

```

Comment:
What do we think of this name?

Reply:
It specifically parses the charts in the spreadsheet, so I would rename to `SpreadsheetChartParser`

Reply:
Updated to `ExcelChartParser`